src/osd/scrubber/pg_scrubber.cc

```diff
@@ -127,7 +127,7 @@ bool PgScrubber::verify_against_abort(epoch_t epoch_to_verify)
 	   << " vs last-aborted: " << m_last_aborted << dendl;
 
   // if we were not aware of the abort before - kill the scrub.
-  if (epoch_to_verify > m_last_aborted) {
+  if (epoch_to_verify >= m_last_aborted) {
```

Comment:
It isn't strictly necessary to incorporate both of these changes (L130 and L149) to rectify the specific scenario that we witnessed, and that I laid out in the associated tracker. I included this because it seems harmless to do so, and because it provides a safeguard against other unknown and unexpected cases where `should_abort` returns true twice within the same epoch. Since the PG lock is held while this method is invoked, I didn't see how it would actually matter if the message is from the same or an older epoch relative to m_last_aborted - in any case, if should_abort returns true, it is relevant to abort the currently active scrub.